lib/index.js

```diff
@@ -31,8 +31,20 @@ export default {
     this.subscriptions.add(atom.config.observe('flow-ide.onlyIfAppropriate', (onlyIfAppropriate) => {
       this.onlyIfAppropriate = onlyIfAppropriate
     }))
+
+    let priority = atom.config.get('flow-ide.hyperclickPriority')
```

Comment:
Actually, the first version of the code had better behavior, since it would only ask for a restart if the setting changed from the initial value. Your version would prompt on every change unless the
 user hadn't yet removed the previous notification 😛.

Something like this should combine both:
```js
let initialValue = null
let restartNotification
this.hyperClickPriority = null
this.subscriptions.add(atom.config.observe('flow-ide.hyperClickPriority', (hyperClickPriority) => {
  if (initialValue === null) {
    initialValue = hyperClickPriority
  }
  if (this.hyperClickPriority !== initialValue) {
    restartNotification = atom.notifications.addSuccess('Restart atom to update flow-ide priority?', {
      dismissable: true,
      buttons: [{
        text: 'Restart',
        onDidClick: () => atom.restartApplication(),
      }],
    })
  } else if (restartNotification) {
    restartNotification.dismiss()
  }
  this.hyperclickPriority = hyperclickPriority
}))
```